### DISCUSSION TITLE
RequestForComment - Enhancement common.states

### DISCUSSION THREAD

**Discussion by Apollon77** [Zefau/ioBroker.jarvis#1753] [Ideas / Feature Requests]
Hi @Zefau,

we plan to enhance the definition of common.states  definitoipn as described in https://github.com/ioBroker/ioBroker.docs/pull/387/files

Please provide feedback and also an info if you would support this as adjustment in your adapter in the future.

Thank you

Ingo

**Comment by Zefau** [comment]
@Apollon77 sorry for my late reply. Following the link it looks like it has already been merged?

From what I read I can adapt the newly introduced features, but it is not required?
Do you have examples to go through?

**Reply by Apollon77** [reply]
Hi, yes after some feedback and discussion on dev meeting we decided to allow it. 

In fact Inhaber no real world example right now because I assume no adapter uses that so far. Sorry.
